activerecord/CHANGELOG.md

```diff
@@ -23,5 +23,9 @@
 
     *DHH*
 
+*   Normalize the date component to account for DST.
```

Comment:
New changelog entires are on the top of the file

Reply:
FIxed - thanks!